check_api/src/main/java/com/google/errorprone/ErrorProneOptions.java

```diff
@@ -412,4 +435,12 @@ public static ErrorProneOptions processArgs(String[] args) {
     Preconditions.checkNotNull(args);
     return processArgs(Arrays.asList(args));
   }
+
+  private static Pattern getExcludedPattern(Iterable<String> excludeSubStrings) {
```

Comment:
WDYT about making the flag take a regex, instead of a comma-separated list that gets processed into a regex?

Reply:
Makes sense; will change